src/sagemaker/jumpstart/accessors.py

```diff
@@ -78,6 +78,22 @@ def _set_cache_and_region(region: str, cache_kwargs: dict) -> None:
             )
             JumpStartModelsAccessor._curr_region = region
 
+    @staticmethod
+    def get_manifest(region: str = JUMPSTART_DEFAULT_REGION_NAME) -> List[JumpStartModelHeader]:
+        """Return entire JumpStart models manifest.
+
+        Raises:
+            ValueError: If region in `cache_kwargs` is inconsistent with `region` argument.
+
+        Args:
+            region (str): Optional. The region to use for the cache.
+        """
+        cache_kwargs = JumpStartModelsAccessor._validate_and_mutate_region_cache_kwargs(
+            JumpStartModelsAccessor._cache_kwargs, region
+        )
+        JumpStartModelsAccessor._set_cache_and_region(region, cache_kwargs)
+        return JumpStartModelsAccessor._cache.get_manifest()  # type: ignore
```

Comment:
Is this backward compatible change. looks like you removed a variable from this function.

Reply:
Good call-out. This isn't backward compatible. However, this function is only invoked internally and is not referenced in any public documentation or notebooks. So I don't think this is a risk. But thank you for raising my awareness of these issues.

Reply:
if it's internal then function should start with `_` as convention. Can you please change it. 
Also it's a static method so anyone from outside can use it. can you please remove static decorator from it.

Reply:
Sorry, this isn't internal. I will callout this is not backwards compatible in the PR description.

Reply:
I am not comfortable merging a backward incompatible change, you can have it none by default. if it’s not passed then you can get that value.

Reply:
Got it. I have made this implementation backwards compatible.